### PR TITLE
ARK-45 Elements Mission

### PR DESCRIPTION
**Pull request by GeronimoJohn**
- Create the elements/mission component
- Do not add the background photo as seen on the JFP Figma
- Merge the main branch to this branch to ensure its updated
- replaces https://github.com/JesusFilm/Ark/pull/38

### CODE REVIEW

File: src/elements/mission/mission.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { BasicMission } from './mission.composition'
+
+it('should render with the mission text', () => {
+  const { getByText } = render(<BasicMission />)
```
Comment (tataihono) [inline src/elements/mission/mission.spec.tsx:6]: tests should avoid using compositions since their underlying data can change causing the test to break.

**Review by tataihono — CHANGES_REQUESTED** [review]
hmm. most of this component needs to be refactored sorry.

File: src/elements/mission/mission.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  },
+  thirdDivider: {
+    marginTop: theme.spacing(6),
+    marginBottom: theme.spacing(0.5)
+  },
+  fourthDivider: {
+    marginBottom: theme.spacing(5)
+  },
+  textColor: {
+    color: '#EF3340'
```
[the remark this answers is not in the document]
Comment (GeronimoJohn) [reply src/elements/mission/mission.tsx:20]: I did but theme.palette.primary.main is giving back the color blue instead. Though in the theme.ts file it is set to the red that we want.

Screen Shot 2021-07-19 at 10 42 08 AM

File: src/elements/mission/mission.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  fourthDivider: {
+    marginBottom: theme.spacing(5)
+  },
+  textColor: {
+    color: theme.palette.primary.main
+  }
+}))
+
+export function Mission() {
+  const { t } = useTranslation()
```
Comment (tataihono) [inline src/elements/mission/mission.tsx:26]: add a namespace here for now.
```suggestion
  const { t } = useTranslation('mission')
```

**Review by tataihono — DISMISSED** [review]

File: src/elements/mission/mission.docs.mdx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import { Mission } from './mission';
+
+## React Component that renders the mission text
+A component that renders the mission outline
+
+### Using props to customize the text
+
+Modify the text to see it change live:
+```js live
+<Mission text="Together, we will share Jesus with 5 billion people by 2025. This is where you can read and share stories of how God's amazing love and power is reaching people through churches, organizations, Bible studies, technology and more." />
```
Comment (tataihono) [inline src/elements/mission/mission.docs.mdx:15]:
```suggestion
<Mission />
```

File: src/elements/mission/mission.docs.mdx [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
+description: 'A Mission component.'
+labels: ['React', 'typescript', 'block']
```
Comment (tataihono) [inline src/elements/mission/mission.docs.mdx:3]:
```suggestion
labels: ['React', 'Typescript', 'Mission']
```

**Review by tataihono — CHANGES_REQUESTED** [review]

**Review by tataihono — APPROVED** [review]

**Pull request merged**